acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -63,8 +73,13 @@ def __init__(self,
         self.energy_axis = energy_axis
         self.max_offset = max_offset
         self.exclude_regions = exclude_regions
-        self.min_observation_per_cos_zenith_bin = min_observation_per_cos_zenith_bin
+        self.cos_zenith_binning_method = cos_zenith_binning_method
+        self.livetime_condition = "livetime" in cos_zenith_binning_method
+        self.wobble_condition = "wobble" in cos_zenith_binning_method
```

Comment:
Weird usage of `cos_zenith_binning_method`. You define 4 possible values but then ignore them for partial keyword searches. 
Here "livetimewobble" is valid and True for both. Same result with "min_n_observation_not_livetime_ignore_wobble".

This needs to be changed.

Reply:
Now there is a check if the method's full string is found in an array with all the possible methods.
The booleans are now based on the array index of the method.
This is done directly in the cos zenith binning method